parser.js

```diff
@@ -800,7 +800,7 @@ class Parser extends EventEmitter {
   }
 
   _emitError (err) {
-    debug('_emitError')
+    debug('_emitError', err)
```

Comment:
This is needed because when looking for this error I had no clue what was going on even by enabling debug, I just saw an emitError log that doens't say anything about why it's failing